test/extended/builds/remove_buildconfig.go

```diff
@@ -0,0 +1,68 @@
+package builds
+
+import (
+	"time"
+
+	"k8s.io/kubernetes/pkg/util/wait"
+
+	g "github.com/onsi/ginkgo"
+	o "github.com/onsi/gomega"
+
+	exutil "github.com/openshift/origin/test/extended/util"
+)
+
+var _ = g.Describe("builds: deleting buildconfig", func() {
+	defer g.GinkgoRecover()
+	var (
+		buildFixture = exutil.FixturePath("..", "extended", "fixtures", "test-build.json")
+		oc           = exutil.NewCLI("cli-start-build", exutil.KubeConfigPath())
+	)
+
+	g.JustBeforeEach(func() {
+		g.By("waiting for builder service account")
+		err := exutil.WaitForBuilderAccount(oc.KubeREST().ServiceAccounts(oc.Namespace()))
+		o.Expect(err).NotTo(o.HaveOccurred())
+		oc.Run("create").Args("-f", buildFixture).Execute()
+	})
+
+	g.Describe("oc delete buildconfig", func() {
+		g.It("should start a build and wait for the build to complete", func() {
```

Comment:
"oc delete buildconfig should start a build and wait for the build to complete" ?!

I thought it should delete a buildconfig.